README.md

```diff
@@ -45,6 +45,8 @@ This document borrows heavily from:
 * Formats should be in the form of api/v2/resource/{id}
 * Design resource representations. Don’t simply represent database tables.
 * Support field projections on resources. Allow clients to reduce the number of fields that come back in the response.
+* For non-resource requests, url should identify the action being taken. E.g. if the endpoint converts 100 euros to chinese yen, is should look like `/convert?from=EURO&to=YEN&amount=100`
+* Make it clear in your API documentation that the "non-resource" scenarios are different
```

Comment:
@snamoah Please how do you make it clear that "non-resource" scenarios are different in your API. Can you site an example?

And can you give an example of an endpoint that accepts non-resource POST requests if possible?

Reply:
@richardkdbz the example exists in the guidelines. is it not clear enough?

Reply:
@snamoah I mean example on how you make it clear that "non-resource" scenarios are fiferent.

Reply:
@richardkdbz upated